openjdk.hcl

```diff
@@ -87,6 +87,20 @@ version "17.0.2_8" {
   }
 }
 
+version "17.0.5" {
```

Comment:
FYI I changed this to 17.0.5_8

Reply:
Ah, right, I missed that precedent in this file. 👍